.github/workflows/ci.yml

```diff
@@ -436,7 +436,7 @@ jobs:
           echo '::group::Compiling testsuite'
           ./mvnw clean install -nsu -B -Pauth-server-quarkus -DskipTests -f testsuite/pom.xml
           echo '::endgroup::'
-          ./mvnw clean install -nsu -B -Pauth-server-cluster-quarkus -Dsession.cache.owners=2 -Dtest=**.cluster.** -f testsuite/integration-arquillian/pom.xml  | misc/log/trimmer.sh
+          ./mvnw clean install -nsu -B -Pauth-server-cluster-quarkus -Dsession.cache.owners=2 -Dtest=**.cluster.** -f testsuite/integration-arquillian/tests/base/pom.xml  | misc/log/trimmer.sh
```

Comment:
This is to make surefire happy. Without this change it tries to run `**.cluster.**` tests inside all `integration-arquillian` modules, even those without tests, so it fails.